marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/StepImpl.java

```diff
@@ -26,6 +26,8 @@
 import java.util.Map;
 
 public class StepImpl implements Step {
+    public final static int DEFAULT_BATCH_SIZE = 100;
+    public final static int DEFAULT_THREAD_COUNT = 4;
```

Comment:
We don't need defaults as we fallback on the properties set in the Flow.

Reply:
Fixed in the latest commit